.github/workflows/dotnet.yml

```
@@ -0,0 +1,43 @@
+        submodules: recursive
+    - name: Setup .NET
+      uses: actions/setup-dotnet@v2
+      with:
+        dotnet-version: 8.0.x
+    - name: Restore dependencies
+      run: dotnet restore
+    - name: Build
+      run: dotnet build --no-restore -c Release
+    - name: Test
+      run: dotnet test --no-build --verbosity normal -c Release
+    # - name: Pack
```

yamllint

[warning] comment not indented like content

(comments-indentation)